ponder.config.ts

```diff
@@ -1,29 +1,28 @@
-import { ACTIVE_PLUGIN } from "./src/lib/plugin-helpers";
+import { deepMergeRecursive } from "./src/lib/helpers";
+import { type IntersectionOf, getActivePlugins } from "./src/lib/plugin-helpers";
 import * as baseEthPlugin from "./src/plugins/base.eth/ponder.config";
 import * as ethPlugin from "./src/plugins/eth/ponder.config";
 import * as lineaEthPlugin from "./src/plugins/linea.eth/ponder.config";
 
 const plugins = [baseEthPlugin, ethPlugin, lineaEthPlugin] as const;
```

Comment:
```suggestion
const availablePlugins = [baseEthPlugin, ethPlugin, lineaEthPlugin] as const;
```

Suggesting this to help us give more distinct naming to all available plugins vs all activated plugins.